libs/langchain/langchain/smith/evaluation/runner_utils.py

```diff
@@ -963,10 +963,12 @@ def _prepare_eval_run(
     project_name: str,
     project_metadata: Optional[Dict[str, Any]] = None,
     tags: Optional[List[str]] = None,
+    dataset_version: Optional[Union[str, datetime]] = None,
```

Comment:
what happens if it's a string?

Reply:
Version tags ("prod")

But it's ambiguous what would happen if you have a string datetime or something, which is bad